musicbot/bot.py

```diff
@@ -1311,8 +1311,11 @@ def _gen_embed(self):
                         return Response(self.str.get('cmd-play-spotify-album-queued', "Enqueued `{0}` with **{1}** songs.").format(res['name'], len(res['tracks']['items'])))
                     elif 'playlist' in parts:
                         res = await self.spotify.get_playlist(parts[-3], parts[-1])
+                        while int(res["tracks"]["total"]) > len(res['tracks']['items']):
+                            resp = await self.spotify.get_playlist(parts[-3], parts[-1], offset=len(res['tracks']['items']))
+                            res['tracks']['items'].extend(resp['tracks']['items'])
                         await self._do_playlist_checks(permissions, player, author, res['tracks']['items'])
-                        procmesg = await self.safe_send_message(channel, self.str.get('cmd-play-spotify-playlist-process', 'Processing playlist `{0}`').format(res['name']))
+                        procmesg = await self.safe_send_message(channel, self.str.get('cmd-play-spotify-playlist-process', 'Processing playlist `{0} ({1})`').format(res['name']), song_url)
```

Comment:
Should be 'Processing playlist `{0}` (`{1}`)', the URI should not look like part of the song title